integration-cli/cli/cli.go

```diff
@@ -30,6 +30,10 @@ type testingT interface {
 	Fatalf(string, ...interface{})
 }
 
+type TestingT interface {
```

Comment:
Yes because it is used in definition of `dockerCmd` which is used in both benchmarks and tests. If I replace it with `*testing.T` I have this error:
```
integration-cli/docker_cli_logs_bench_test.go:11:21: cannot use c (type *testing.B) as type *testing.T in argument to dockerCmd
```